financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -19,20 +19,30 @@ import androidx.navigation.compose.rememberNavController
 import com.airbnb.mvrx.MavericksView
 import com.airbnb.mvrx.viewModel
 import com.airbnb.mvrx.withState
+import com.stripe.android.financialconnections.di.FinancialConnectionsSheetNativeComponent
 import com.stripe.android.financialconnections.features.consent.ConsentScreen
 import com.stripe.android.financialconnections.features.institutionpicker.InstitutionPickerScreen
 import com.stripe.android.financialconnections.navigation.NavigationDirections
+import com.stripe.android.financialconnections.navigation.NavigationManager
 import com.stripe.android.financialconnections.presentation.CreateBrowserIntentForUrl
 import com.stripe.android.financialconnections.presentation.FinancialConnectionsSheetNativeViewEffect.OpenAuthFlowWithUrl
 import com.stripe.android.financialconnections.presentation.FinancialConnectionsSheetNativeViewModel
 import com.stripe.android.financialconnections.ui.theme.FinancialConnectionsTheme
+import javax.inject.Inject
 
 internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity(), MavericksView {
 
     val viewModel: FinancialConnectionsSheetNativeViewModel by viewModel()
 
+    @Inject
+    lateinit var navigationManager: NavigationManager
+
+    val activityRetainedComponent: FinancialConnectionsSheetNativeComponent
```

Comment:
nit: use `activityRetainedComponent.inject` at line 45.

Or if you define the subcomponent builders as @Inject fields like `navigationManager`,  there is no need to hold `activityRetainedComponent` in the activity, instead you could call `viewModelContext.activity<FinancialConnectionsSheetNativeActivity>().consentBuilder` and `viewModelContext.activity<FinancialConnectionsSheetNativeActivity>().institutionPickerBuilder` directly